src/incubator/TextField/CharCounter.tsx

```diff
@@ -0,0 +1,32 @@
+import React, {useContext} from 'react';
+import {TextStyle, StyleSheet} from 'react-native';
+import _ from 'lodash';
+import Text from '../../components/text';
+import FieldContext from './FieldContext';
+
+export interface CharCounterProps {
+  showCharCounter?: boolean;
```

Comment:
Why do you need this prop here? If it was passed to TextField you render it and if not you don't, no?

Reply:
Are you talking about the `maxLength` prop, right? 
I need to declare it here so it's recognized (typescript wise) when I use it in the component.